low-level/files/opt/seagate/sspl/setup/resource_map/server.py

```diff
@@ -44,10 +44,12 @@ def __init__(self):
         self.sysfs.initialize()
         self.sysfs_base_path = self.sysfs.get_sysfs_base_path()
         self.cpu_path = self.sysfs_base_path + CPU_PATH
-        self.server_frus = {
+        self.server_map = {
             'cpu': self.get_cpu_info,
             'platform_sensors': self.get_platform_sensors_info,
-            'memory': self.get_mem_info
+            'memory': self.get_mem_info,
+            'sas_hba': self.get_sas_hba_info,
+            'sas_ports': self.get_sas_ports_info,
```

Comment:
',' after last element shouldnt exist

Reply:
removed 👍 , had kept it as it makes it easy while rebasing or merging the PR's.